example/test.js

```diff
@@ -0,0 +1,10 @@
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
```

Comment:
// Would prefer the public way of using it for the purpose of education.
var cssToJss = require('..').cssToJss